packages/pipeline/test/parsers/etherscan/index_test.ts

```diff
@@ -0,0 +1,65 @@
+import { BigNumber } from '@0x/utils';
+import * as chai from 'chai';
+import 'mocha';
+
+import { EtherscanTransactionResponse } from '../../../src/data_sources/etherscan';
+import { EtherscanTransaction } from '../../../src/entities';
+import { parseEtherscanTransactions } from '../../../src/parsers/etherscan';
+import { chaiSetup } from '../../utils/chai_setup';
+
+chaiSetup.configure();
+const expect = chai.expect;
+
+// tslint:disable:custom-no-magic-numbers
+describe('etherscan_transactions', () => {
+    describe('parseEtherscanTransactions', () => {
+        it('converts etherscanTransactions to EtherscanTransaction entities', () => {
+            const response: EtherscanTransactionResponse[] = [
```

Comment:
can we use an actual API response for the test? Then it will also test if our defined type for the API response works as expected. 

There is an example of how we do this in the Crypto Compare scripts.

I think it would be an improvement, but I don't feel strongly on this.

Reply:
I've updated it now to use `EtherscanResponse` - was this what you had in mind?

Reply:
Sorry, I pointed you to the wrong example before. It's in [Copper scripts](https://github.com/0xProject/0x-monorepo/blob/development/packages/pipeline/test/parsers/copper/index_test.ts). I actually meant using the raw  response from the API. I think it's a slightly more useful test than predefining a typescript object.

The type conversion should automatically catch any errors, but the approach with raw JSON fixtures means if anything breaks, we can quickly check for API compatibility by doing a `curl` and dumping the results in the fixtures files.

Up to you if you think it's worth implementing ^^

Reply:
Ah got it! Thanks - I'll update :)

Reply:
This was a really good call - it helped me surface a small bug. Thanks @xianny ! 🙏